dev/reference/performance/statistics.md

```diff
@@ -229,7 +229,7 @@ SHOW STATS_BUCKETS [ShowLikeOrWhere];
 | db_name  |  数据库名    |
 | table_name | 表名 |
 | partition_name | 分区名 |
-| column_name | 列名 |
+| column_name | 根据 is_index 来变化：is_index 为 0 时是列名，为 1 时是索引名 |
```

Comment:
Update the `column_name` in `show stats_histograms`?

Reply:
Updated.